nautobot/utilities/templates/utilities/templatetags/dynamic_filter_form_modal.html

```diff
@@ -0,0 +1,52 @@
+{% load form_helpers %}
+{% load static %}
+
+<div class="modal fade" tabindex="-1" id="{{ form_name }}_modal">
+    <div class="modal-dialog modal-lg" style="width: 90% !important;">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Filter Form</h4>
+            </div>
+            <div class="modal-body">
+                {% if form %}
+                    <form id="dynamic-filter-form" class="form-horizontal" method="get" action=".">
+                        <div class="">
+                            <table class="table" id="secrets" style="table-layout: fixed;">
+                                {{ form.management_form }}
+                                {% for form in form %}
+                                    {% if forloop.first %}
+                                        <thead>
+                                            <tr>
+                                                {% for field in form.visible_fields %}
+                                                    <th>{{ field.label|capfirst }}</th>
+                                                {% endfor %}
+                                            </tr>
+                                        </thead>
+                                    {% endif %}
+                                    <tr class="formset_row-dynamic-filterform">
+                                        {% for field in form.visible_fields %}
+                                            <td>
+                                                {{ field }}
+                                            </td>
+                                        {% endfor %}
+                                    </tr>
+                                {% endfor %}
+                            </table>
+                        </div>
+
+                        <div class="text-right noprint">
+                            <button type="submit" class="btn btn-info">
+                                <span class="mdi mdi-magnify" aria-hidden="true"></span> Apply
+                            </button>
+                            <a href="." class="btn btn-default">
+                                <span class="mdi mdi-close-thick" aria-hidden="true"></span> Clear
```

Comment:
Just checking - does this close the form (and revert to any previously applied filters) or does it actually clear all filters that may have been applied by a previous dialog?

Reply:
It close the form (and revert to any previously applied filters)

Reply:
OK, so maybe the button should be "Cancel" rather than "Clear"?